extensions/amp-megaphone/0.1/test/validator-amp-megaphone.out

```diff
@@ -0,0 +1,101 @@
+|      <script async src="https://cdn.ampproject.org/v0.js"></script>
+|  </head>
+|  <body>
+|    <!-- valid: classic mode -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951"
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: tile mode -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-tile="true"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:36:2 The attribute 'data-tile' in tag 'amp-megaphone [data-episode]' is set to the invalid value 'true'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: light mode -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-light="true"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:39:2 The attribute 'data-light' in tag 'amp-megaphone [data-episode]' is set to the invalid value 'true'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: integer start time -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-start="56"
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: decimal start time -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-start="300.32"
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- invalid: bad start time -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-start="300s"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:48:2 The attribute 'data-start' in tag 'amp-megaphone [data-episode]' is set to the invalid value '300s'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- invalid: light mode must be 'true' or 'false'-->
+|    <amp-megaphone height=166 data-trackid="243169232" data-light
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:51:2 The tag 'amp-megaphone' is disallowed except in specific forms. [GENERIC]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- invalid: bad episode id-->
+|    <amp-megaphone height=166 data-episode="mahler_number_6.ogg"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:54:2 The attribute 'data-episode' in tag 'amp-megaphone [data-episode]' is set to the invalid value 'mahler_number_6.ogg'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- invalid: both episode and playlist specified-->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-playlist="DEM6640968282"
+|                    layout="fixed-height"></amp-megaphone>
```

Comment:
This does not produce an error. Should it?	 It might be hard since any `data-` attributes are generally allowed wherever.